### PR TITLE
Update MPAS website for E3SM MPAS transplant

### PR DESCRIPTION
**Pull request by mark-petersen**
Update text at https://mpas-dev.github.io/ for ocean and sea ice transplant.

### CODE REVIEW

**Comment by mark-petersen** [comment]
@xylar @akturner @matthewhoffman you are welcome to grab this branch and open index.html in a web browser. Or, for the simple version read on:

**Comment by mark-petersen** [comment]
@xylar please review this text for the ocean:

As of May 2021, MPAS-Ocean is developed within the E3SM repository at https://github.com/E3SM-Project/E3SM. The code is located in the subdirectory components/mpas-ocean, and can be compiled as a stand-alone ocean model, or within the E3SM coupled climate model. We also support the Configuration Of Model for Prediction Across Scales Setups (COMPASS) repository at https://github.com/MPAS-Dev/compass. COMPASS is a python package that provides an automated system to set up test cases for MPAS-Ocean. There is further documentation on both COMPASS and running MPAS-Ocean at https://mpas-dev.github.io/compass.

Previous releases of MPAS-Ocean are listed below... 

Then I left everything else the same as before, including the release table.

**Comment by mark-petersen** [comment]
@akturner please review, I simply added this at the top of the sea ice release page:

As of May 2021, MPAS-Seaice is developed within the E3SM repository at https://github.com/E3SM-Project/E3SM. The code is located in the subdirectory components/mpas-seaice, and can be compiled as a stand-alone sea ice model, or within the E3SM coupled climate model.

**Comment by mark-petersen** [comment]
@matthewhoffman because MALI development is a bit different, I wasn't sure what to put. I could put the identical thing:

As of May 2021, MPAS-Albany-Landice is developed within the E3SM repository at https://github.com/E3SM-Project/E3SM. The code is located in the subdirectory components/mpas-albany-landice, and can be compiled as a stand-alone land ice model, or within the E3SM coupled climate model.

You are welcome to edit this html directly, or just give me some text here as a comment.

**Comment by matthewhoffman** [comment]
@mark-petersen , thanks for doing this.  I'll specify a slightly different version for MALI to account for our primary development repo:

As of May 2021, MPAS-Albany-Landice is developed within the E3SM repository.  MALI development occurs in a development repository at https://github.com/MALI-Dev/E3SM.  (Development in that repository is periodically merged into the primary E3SM repository, https://github.com/E3SM-Project/E3SM.)  The MALI code is located in the subdirectory components/mpas-albany-landice, and can be compiled as a stand-alone land ice model, or within the E3SM coupled climate model.

File: ocean/ocean.html [hunk 1]
```diff
@@ -51,7 +51,19 @@
...
+		  <p>Please reference the following manuscripts when introducing MPAS-Ocean: 
+          <ul>
+  <li>
+          Ringler, T., Petersen, M., Higdon, R. L., Jacobsen, D., Jones, P. W., &amp; Maltrud, M. (2013). Ocean Modelling. Ocean Modelling, 69(C), 211–232. doi:10.1016/j.ocemod.2013.04.010 (<a href="http://www.sciencedirect.com/science/article/pii/S1463500313000760" title="A multi-resolution approach to global ocean modeling" target="_new">pdf</a>)
+  </li>
+  <li>
+Petersen, M. R., Asay‐Davis, X. S., Berres, A. S., Chen, Q., Feige, N., Hoffman, M. J., et al. (2019). An evaluation of the ocean and sea ice climate of E3SM using MPAS and interannual CORE‐II forcing. Journal of Advances in Modeling Earth Systems, 11, 1438– 1458. <a href="https://doi.org/10.1029/2018MS001373">https://doi.org/10.1029/2018MS001373</a>
```
Comment (xylar) [inline ocean/ocean.html:60]:
```suggestion
          Petersen, M. R., Asay‐Davis, X. S., Berres, A. S., Chen, Q., Feige, N., Hoffman, M. J., et al. (2019). An evaluation of the ocean and sea ice climate of E3SM using MPAS and interannual CORE‐II forcing. Journal of Advances in Modeling Earth Systems, 11, 1438– 1458. <a href="https://doi.org/10.1029/2018MS001373">https://doi.org/10.1029/2018MS001373</a>
```

File: ocean/ocean.html [hunk 2]
```diff
@@ -51,7 +51,19 @@
...
+  </li>
+  <li>
+Golaz, J.‐C., Caldwell, P. M., Van Roekel, L. P., Petersen, M. R., Tang, Q., Wolfe, J. D., et al. (2019). The DOE E3SM coupled model version 1: Overview and evaluation at standard resolution. Journal of Advances in Modeling Earth Systems, 11, 2089– 2129. <a href="https://doi.org/10.1029/2018MS001603">https://doi.org/10.1029/2018MS001603</a>
```
Comment (xylar) [inline ocean/ocean.html:63]:
```suggestion
          Golaz, J.‐C., Caldwell, P. M., Van Roekel, L. P., Petersen, M. R., Tang, Q., Wolfe, J. D., et al. (2019). The DOE E3SM coupled model version 1: Overview and evaluation at standard resolution. Journal of Advances in Modeling Earth Systems, 11, 2089– 2129. <a href="https://doi.org/10.1029/2018MS001603">https://doi.org/10.1029/2018MS001603</a>
```

File: ocean/releases.html [hunk 3]
```diff
@@ -43,7 +43,9 @@
             <H1>MPAS-Ocean Releases</H1>
 		<div id="apDiv3">
             <br />
-			<p>MPAS-Ocean releases are listed below. Please ensure you are using
+<p>As of May 2021, MPAS-Ocean is developed within the E3SM repository at <a href="https://github.com/E3SM-Project/E3SM">https://github.com/E3SM-Project/E3SM</a>. The code is located in the subdirectory components/mpas-ocean, and can be compiled as a stand-alone ocean model, or within the E3SM coupled climate model. We also support the Configuration Of Model for Prediction Across Scales Setups (COMPASS) repository at <a href="https://github.com/MPAS-Dev/compass">https://github.com/MPAS-Dev/compass</a>. COMPASS is a python package that provides an automated system to set up test cases for MPAS-Ocean. There is further documentation on both COMPASS and running MPAS-Ocean at <a href="https://mpas-dev.github.io/compass">https://mpas-dev.github.io/compass</a>.
```
Comment (xylar) [inline ocean/releases.html:46]:
```suggestion
			<p>As of May 2021, MPAS-Ocean is developed within the E3SM repository at <a href="https://github.com/E3SM-Project/E3SM">https://github.com/E3SM-Project/E3SM</a>. The code is located in the subdirectory components/mpas-ocean, and can be compiled as a stand-alone ocean model, or within the E3SM coupled climate model. We also support the Configuration Of Model for Prediction Across Scales Setups (COMPASS) repository at <a href="https://github.com/MPAS-Dev/compass">https://github.com/MPAS-Dev/compass</a>. COMPASS is a python package that provides an automated system to set up test cases for MPAS-Ocean. There is further documentation on both COMPASS and running MPAS-Ocean at <a href="https://mpas-dev.github.io/compass">https://mpas-dev.github.io/compass</a>.
```

Comment (xylar) [reply ocean/releases.html:46]: I'd break this up into multiple lines for readability, like below.

File: sea_ice/releases.html [hunk 4]
```diff
@@ -16,6 +16,7 @@
   </tbody>
 </table>
 
+<p>As of May 2021, MPAS-Seaice is developed within the E3SM repository at <a href="https://github.com/E3SM-Project/E3SM">https://github.com/E3SM-Project/E3SM</a>. The code is located in the subdirectory components/mpas-seaice, and can be compiled as a stand-alone sea ice model, or within the E3SM coupled climate model.
```
Comment (xylar) [inline sea_ice/releases.html:19]: Again, it would be easier to read and edit if this were multiple lines

**Review by xylar — APPROVED** [review]
I have a few super nit-picky comments about formatting that you can take or leave but the content looks great to me.

Longer term, it would be great to have someone really put some time into making a slicker, more professional looking website but that time is not now.

**Comment by mark-petersen** [comment]
@matthewhoffman this is what this page looks like now. Feel free to edit further.
image

**Comment by mark-petersen** [comment]
@akturner Here is the sea ice page. Feel free to edit further, or approve this PR.
image

**Comment by xylar** [comment]
@mark-petersen, I realize this is mission creep but would you be willing to fix the bad link (from 8 years ago!) here?https://github.com/MPAS-Dev/mpas-dev.github.com/blob/master/README.md
It should be mpas-dev.github.io, not mpas-dev.github.com.

Also, do we do anything with the information when folks register before downloading the code?  If no, I would suggest we reconsider that step.  There are a ton of different ways folks are likely to download the code without registering (or even knowing that there is a place to register) so I wonder why we would want this impediment to downloading the code in this particular way.  When I come across such a request for other codes, I usually perceive it as super antiquated, ignore it, and just download the code.

**Comment by mark-petersen** [comment]
Good point. I thought the same thing about the registration. I'll change it. BTW 'antiquated' is a great word.

**Comment by mark-petersen** [comment]
@matthewhoffman this is the current landice page, with the registration link removed. Let me know if you have further comments.
image

**Review by matthewhoffman — APPROVED** [review]
Thanks, Mark.  This looks good.

**Comment by akturner** [comment]
@mark-petersen : Looks great! Thanks for making these changes!

**Pull request merged**